JS/Planes/PassengerPlane.js

```diff
@@ -2,21 +2,18 @@ const Plane = require('./Plane');
 
 class PassengerPlane extends Plane {
 
-    //=================CONSTRUCTORS=================
-
     constructor(model, maxSpeed, maxFlightDistance, maxLoadCapacity, passengersCapacity) {
         super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
-        this._passengersCapacity = passengersCapacity;
+        this.passengersCapacity = passengersCapacity;
     }
-
-    //=================METHODS=================
+    
     getPassengersCapacity() {
-        return this._passengersCapacity;
+        return this.passengersCapacity;
     }
 
-    // set passengersCapacity(value) {
-    //     this._passengersCapacity = value;
-    // }
+    setPassengersCapacity(value) {
+        this.passengersCapacity = value;
+    }
 }
 
 module.exports = PassengerPlane;
```

Comment:
The best practice is to use the following syntax while exporting variables from the file:
`module.exports = { PassengerPlane }`

Reply:
Done